extension.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <title>PointerEvents - Level 2 - Extensions</title>
+    <link rel="stylesheet" type="text/css" href="https://w3c.github.io/uievents/spec-WD.css">
+    <script src='https://www.w3.org/Tools/respec/respec-w3c-common.js' class='remove'></script>
+    <script class='remove'>
+      var respecConfig = {
+        specStatus: "CG-DRAFT",
```

Comment:
I think you're missing something in the metadata here.  The [status](https://rawgit.com/NavidZ/pointerevents/b9f39bd6a3d1011436a45745035a2d0d54be3838/extension.html#sotd) says "This specification was published by the .".  You can use "Pointer Events Working Group" for now (eventually we'll have a CG to put here).